src/Api/Controllers/OrganizationUsersController.cs

```diff
@@ -87,6 +89,33 @@ public async Task<IEnumerable<string>> GetGroups(string orgId, string id)
             var responses = groupIds.Select(g => g.ToString());
             return responses;
         }
+        
+        [HttpGet("{userId}/reset-password-details")]
```

Comment:
shouldn't this be `id`, as in "organization user id" like other endpoints in this controller?

Reply:
It /can/ be. But I'm actually using Organization User - User ID to make the data layer call, so I changed the descriptor to `userId` to not confuse with `id` as mentioned. Do you have a strong preference?

Reply:
Following RESTful API design, this endpoint is for organization user actions, so I would expect the `organizations/{orgId}/users/{id}/{action}` to always be {id}=OrganizationUser.Id. Using the User.Id creates inconsistency in the API.

Reply:
Updated